test/fixtures/psetv2.json

```diff
@@ -306,5 +306,8 @@
       }
     ]
+  },
+  "edgeCases": {
+    "libwally": "cHNldP8BAgQCAAAAAQMEQI8kZgEEAQEBBQECAfsEAgAAAAABAU4BSZqBhUX2uuOfwDtjfypOHmTlkMrBvDpvbXGqRENlTBQBAAAAAAAAKQQAIgAgGPH0uXuVVsnhtA01POa1JKpqCx1CQyi1sQMP+RX6ibkBAwQBAAAAAQX9DAFkBECPJGaxdSEDKElhTHBSmusfkGSGm5umFyUiX0RXKArB+VS8/0aDGvOtZ4IBRIh4ggEkiGt8fnx+fH58fnx+fH54ggEgiGt8fnyCWIh+qCECd4PiwfZk7Mla2DHfCd6b575yTpqzh5oEkVjg+F+/NYXCggFEiHiCASSIa3x+fH58fnx+fH58fniCASCIa3x+fIJYiH6oIQJ3g+LB9mTsyVrYMd8J3pvnvnJOmrOHmgSRWOD4X781hcJsbGxsUnqIhwCILQFJmoGFRfa645/AO2N/Kk4eZOWQysG8Om9tcapEQ2VMFAEAAAAAAAAB9AABanx+fH6qfn5+fn5+fH6oa25sfMJ8UX58rWhRIgYDKElhTHBSmusfkGSGm5umFyUiX0RXKArB+VS8/0aDGvMQjvpohQAAAAAAAAAAAwAAAAEIAQABDiDK5EoaCg/7BWGprbueRgqwuRhB/b5LEoVvgGFQ3QTWrAEPBAAAAAAAAQMIhSgAAAAAAAABBBYAFIfYnEPW9Jjvqj5oGlXQKsaYztGWB/wEcHNldAIgSZqBhUX2uuOfwDtjfypOHmTlkMrBvDpvbXGqRENlTBQH/ARwc2V0CAQAAAAAAAEDCH8AAAAAAAAAAQQAB/wEcHNldAIgSZqBhUX2uuOfwDtjfypOHmTlkMrBvDpvbXGqRENlTBQH/ARwc2V0CAQAAAAAAA=="
```

Comment:
let's add this to the roundtrip valid cases.. we can name it `libwally compatibility`